website/content/commands/acl/auth-method/create.mdx

```diff
@@ -80,7 +76,11 @@ Usage: `consul acl auth-method create [options] [args]`
   verified identity attributes returned from the auth method during login to
   determine if the namespace rule applies. Added in Consul 1.8.0.
 
-@include 'http_api_partition_options.mdx'
+#### API Options
```

Comment:
Because not all commands have enterprise options, I decided on the following sequencing:
- Command Options (which are specific to this command)
- Enterprise Options (which apply to a number of commands, but not all ... and have some variation)
- API Options (which apply to every command)

There are other ways we could break this up (e.g., include merge enterprise options into the command options section and use the inline enterprise alert), but I wasn't convinced that would be better.